src/paras/Assignment15/BankUsingSwitch.java

```diff
@@ -0,0 +1,151 @@
+/*
+ Assignment - 15 : 9th April'2021
+
+A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+		                 SBI
+ */
+
+package paras.Assignment15;
+
+public class BankUsingSwitch {
+
+	int iciciTotalBalance, HdfcTotalBalance, bobTotalBalance, sbiTotalBalance;
+	int iciciCredit, hdfcCredit, bobCredit, sbiCredit;
+	int iciciDebit, hdfcDebit, bobDebit, sbiDebit;
+	
+	void setBalance(String bank, int Balance) {
+		switch(bank) {
+		case "ICICI" :
+			iciciTotalBalance = Balance;
+			break;
+		case "HDFC" :
+			HdfcTotalBalance = Balance;
+			break;
+		case "BOB" :
+			bobTotalBalance = Balance;
+			break;
+		case "SBI" :
+			sbiTotalBalance = Balance;
+			break;
+		default:
+			System.out.println("Please Select Proper Bank name");
+		}
+	}
+	
+	void transaction(String operation, String bank, int amount) {
+		switch(operation){
+		case "Debit":
+			switch(bank) {
+			case "ICICI" :
+				if(iciciTotalBalance > amount) {
```

Comment:
instead of writing multi-time same code block you can write debit() method which accept the two arguments and you can simply call that method